### DISCUSSION TITLE
Multiple Executable Instances Running in the Background

### DISCUSSION THREAD

**Discussion by deepaks11** [pyinstaller/pyinstaller#7903] [Help]
I use PyInstaller to generate an "exe" file. However, when I run this executable file, I notice that there are five process instances in the task manager. Do you have any ideas why this might be happening?
![Screenshot ](https://github.com/pyinstaller/pyinstaller/assets/113359279/14ab624b-c19b-4065-880e-592583e1071a)

**Comment by bwoodsend** [comment]
Are you sure you or one of your dependencies aren't just using `multiprocessing` or something equivalent?

**Reply by rokm** [reply]
You can right-click on the header of the list view (the part where it says Name, Status, CPU), and add a "Command Line" column to display command-line arguments of those processes. That would help identifying what kind of processes they are (e.g., if they are `multiprocessing` worker processes, or something else).
